bower.json

```diff
@@ -19,7 +19,7 @@
 		"qunit-assert-close": "JamesMGreene/qunit-assert-close#v1.1.1",
 		"qunit-composite": "JamesMGreene/qunit-composite#v1.1.0",
 		"requirejs": "2.1.14",
-	  	"globalize": "1.0.0",
+	  	"globalize": "globalize#1.1.0-rc.6",
```

Comment:
Nope. Technically the rc.6 is pretty close to 1.1.0. But, writing a blog post about it (for the release) is the biggest uncertainty on my end.